stories/Atom/Buttons-and-labels/Cta_button/buttons.scss

```diff
@@ -0,0 +1,145 @@
+    @include devicebreak(small) {
+        min-width: 168px;
+        display: inline-flex
+    }
+}
+
+.btn-primary {
+    background-color: $color-dark-red;
+    &:hover {
+    background-color: $color-red;
+    }
+}
+
+.btn-secondary {
+    padding: 18px $spacing-24;
+    background-color: $color-blue-600;
+    &:hover {
+        background-color: $color-blue-400;  
+    }
+}
+
+.btn.btn-arrow.disabled {
+    background-color: $color-gray-400;   
+    color: $color-gray-300;
+    pointer-events: none;
+    &::after {
+        content: "";
+        width: 13px;
+        height: 20px;
+        background: url(#{$img-path-Icon}/chevron-right-white.svg)
+        no-repeat left center;
+    }
+}
+
+.btn.btn-no_arrow.disabled, .btn.btn-secondary.disabled {
+    background-color: $color-gray-400;   
+    color: $color-gray-300;
+    pointer-events: none;
+}
+
+.btn-arrow {  
+    &::after {
+        content: "";
+        width: 13px;
+        height: 20px;
+        background: url(#{$img-path-Icon}/chevron-right-black.svg)
+        no-repeat left center;
+        display: block;
+        fill: transparent;
+        cursor: pointer;
+        margin-left: 14px;
+        transition: transform .2s ease;
+    }
+    &:hover::after {
+        background-color: $color-red;
+        transition: transform .2s ease;
+        transform: translateX(70%);
+    }
+}
+
+.btn-no_arrow {
+    padding: 18px $spacing-24;
+    &:hover {
+        background-color: $color-red;
+    }
+}
+
+// CTA arrow and space
+.cta {
+    display: flex;
+    border: none;
+    color: black;
+    font-family: $Font-Family-ProximaRegular;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 1;
+    letter-spacing: 0.03em;
+    align-items: center;
+    @include devicebreak(small) {
+        min-width: 168px;
+        display: inline-flex;
+    }
+
+}
+
+.cta-arrow {
+    &::after {
+        content: "";
+        width: 13px;
+        height: 20px;
+        margin-left: 14px;
+        background: url(#{$img-path-Icon}/arrow-left-expand-defaut.svg)
+        no-repeat left center;
+    }
+    &:hover::after {
+        transition: transform .2s ease;
+        background: url(#{$img-path-Icon}/arrow-left-expand-hover.svg)
+        no-repeat left center;
+        display: block;
+        fill: transparent;
+        cursor: pointer;
+        margin-left: $spacing-24;
+        transform: translateX(10px);
+        transition: transform .2s ease;
+        width: 100px;
+    }
+}
+
+.cta-space {
+    &::after {
+        content: "";
+        width: 13px;
+        height: 20px;
+        background: url(#{$img-path-Icon}/arrow-left-expand-defaut.svg)
+        no-repeat left center;
+        display: block;
+        fill: transparent;
+        cursor: pointer;
+        margin-left: 14px;
+        transition: transform .2s ease;
+    }
+    &:hover::after {
+        transition: transform .2s ease;
+        transform: translateX(70%);
+    }
+}
```

Comment:
@pallavikadam-srijan Please remove unnecessary properties which are not required and please create the placeholders and also keep the styling same for the classes and update the styling only where it is required like cta-arrow can have same styling but just update the hover effects